src/diffusers/models/attention.py

```diff
@@ -375,16 +448,32 @@ def __init__(self, dim_in: int, dim_out: int):
 
         self.proj = linear_cls(dim_in, dim_out * 2)
 
-    def gelu(self, gate: torch.Tensor) -> torch.Tensor:
-        if gate.device.type != "mps":
-            return F.gelu(gate)
-        # mps: gelu is not implemented for float16
-        return F.gelu(gate.to(dtype=torch.float32)).to(dtype=gate.dtype)
-
     def forward(self, hidden_states, scale: float = 1.0):
         args = () if USE_PEFT_BACKEND else (scale,)
         hidden_states, gate = self.proj(hidden_states, *args).chunk(2, dim=-1)
-        return hidden_states * self.gelu(gate)
+        return hidden_states * gelu(gate)
+
+
+# TODO Name?
+class GatedGelu(nn.Module):
```

Comment:
Not super important but do you think it could make sense to include this one under `activations` (along with `ApproximateGELU` from [here](https://github.com/huggingface/diffusers/blob/8dba1808852e7f5c08f91296006ec254cecdd1b1/src/diffusers/models/attention.py#L390))?

Reply:
But I'd understand if you don't want to do it in this PR.

Reply:
removed anyway!